tests/main/views/test_drafts.py

```diff
@@ -2,24 +2,21 @@
 from datetime import datetime
 from flask import json
 import mock
-from app.models import Supplier, ContactInformation, Service, Framework, \
-    DraftService
+from app.models import Supplier, ContactInformation, Service, Framework, DraftService
 from app import db
 
-from nose.tools import assert_equal, assert_in, assert_false
-
 from tests.helpers import FixtureMixin, load_example_listing
 
 
-class TestDraftServices(BaseApplicationTest):
+class TestDraftServices(BaseApplicationTest, FixtureMixin):
     service_id = None
     updater_json = None
     create_draft_json = None
 
     def setup(self):
         super(TestDraftServices, self).setup()
 
-        payload = load_example_listing("G6-IaaS")
+        payload = load_example_listing("G6-SaaS")
```

Comment:
I changed the lot to see what would break, to be honest. And not much does. 
So I figured `SaaS` was more normal, somehow.

There's a bunch of tests that create `Service` objects directly, and when they do they usually do something like